pom.xml

```diff
@@ -78,7 +78,7 @@
 
   <modules>
     <module>declarative-pipeline-migration-assistant-api</module>
-    <module>declarative-pipeline-migration-assistant-plugin</module>
+    <module>declarative-pipeline-migration-assistant</module>
```

Comment:
No required, but I think it makes sense. Both the API plugin and the non-API plugin are plugins, so having one end with plugin is a bit confusing.